controllers/topics.js

```diff
@@ -16,10 +17,15 @@ exports.getTopics = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-    selectArticles()
+    const { topic, sort_by, order } = req.query
+    selectArticles(topic, sort_by, order)
     .then((articles) => {
-        res.send(articles.rows);
-    });
+        articles = articles.rows
+        res.send({articles});
```

Comment:
For consistency's sake, I'd recommend doing the `.rows` refinement always in the model